neqo-http3/src/send_message.rs

```diff
@@ -271,6 +271,10 @@ impl SendStream for SendMessage {
         self.stream.has_buffered_data()
     }
 
+    fn set_sendorder(&mut self, _conn: &mut Connection, _sendorder: Option<SendOrder>) {
+	// Not relevant for SendMessage
```

Comment:
Again.  `self.stream.set_sendorder(conn, sendorder)` seems workable here, even if we don't end up using it.  (Yes, this requires an additional step to implement that on `BufferedStream`, but it's a trivial thing.)

Yes, we're building for webtransport, but this type has a direct relationship with an underlying stream and a random noop is a surprise I'd prefer to avoid.